Source/Fuse.Maps/MapView.uno

```diff
@@ -219,6 +240,16 @@ namespace Fuse.Controls
 			UpdateMarkersNextFrame();
 		}
 
+		void OnPolylineAdded(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
+
+		void OnPolylineRemoved(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
```

Comment:
I don't think there's a good reason for this, but it's not really important either way. I'm fine with keeping it as-is.